### ISSUE TITLE
Mega menu model

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/aria-practices#1013]
I suggest a mega menu version created... I got it going as a rough idea, but I couldn't get the down arrow to continue to work, to open the menu and arrow through the choices. It would require the following modifications to create a mega example.

- Allow the menu sub item to take up full screen width
- Allow something like 3 columns of sub ` <ul> ` elements which have a `<h2>` as the fires `<li>`
- Adjust the JS so that the arrow keys will take user into the sub menu items.

**Comment by DavidMacDonald** [comment]
Note
DIOR did a half decent job of it I think, riffing off the WCAG tutorial 
https://www.dior.com/en_us
choose US.

**Comment by jasonday** [comment]
Isn't this covered by #1070?
https://www.w3.org/TR/wai-aria-practices-1.1/examples/disclosure/disclosure-navigation.html

**Comment by DavidMacDonald** [comment]
@jasonday I don't think so. Mega menus have many characteristics and several designs that are not captured in the generic model disclosure menu.
There are groups of links, sometimes a left panel with details on the right panel like a sideways tabbed interface. Sometimes the parent button at the top is also a link (on hover opens menu on click goes to the category landing page, 
In which case we could provide a link to go to the product category page and next to it a button to open the menu, etc...
And then there is the situation where it collapses into mobile hamburger for all that content.

**Comment by carmacleod** [comment]
@DavidMacDonald There's a "top-level links" variant of the disclosure navigation example currently being reviewed in PR https://github.com/w3c/aria-practices/pull/1614. You can try it out with the Preview link in the opening comment. Feedback welcome.
